assets/styles/parts/utils/_colors.scss

```diff
@@ -6,4 +6,8 @@ All variables are available here:
 https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_colors.scss
 */
 
-$base-primary-color: #FF86D6;
\ No newline at end of file
+$base-primary-color: #FF86D6;
+
+$base-gray-color: #999999;
+$base-light-gray-color: #EAEAEA;
```

Comment:
https://infinum.github.io/eightshift-docs/docs/basics/writing-styles#never-use-the-actual-color-name-as-a-variable-name

Reply:
Is `base-text-muted-color` (instead of `base-gray-color`) and `base-tag-chip-color` (instead of `base-light-gray-color`) ok?

Reply:
Color name should not be related to the implementation. It should be generic as fuck.

So use this site https://chir.ag/projects/name-that-color/#6195ED to generate any color name it really doesn't matter